app/styles/partials/overrides.scss

```diff
@@ -35,16 +35,38 @@ body.dimmable.undetached.dimmed {
   }
 }
 
-.margin.less {
-  padding: 0 !important;
+.less {
+  &.margin {
+    margin: 0 !important;
+  }
+
+  &.bottom.margin {
+    margin-bottom: 0 !important;
+  }
+
+  &.top.margin {
+    margin-top: 0 !important;
+  }
 }
 
 .less {
   &.padding {
     padding: 0 !important;
   }
-  
-  &.padding-right {
+
+  &.right.padding {
     padding-right: 0 !important;
   }
+
+  &.left.padding {
+    padding-left: 0 !important;
+  }
+}
+
+.ui.avatar.image {
+   margin-right: 10px;
+}
+
+.ui.category.search {
```

Comment:
how about not using these default class names at all, and rather making it specific to the page and using !important to enforce the styling?